addons/token-exchange/token_exchanger_agent.go

```diff
@@ -9,6 +9,7 @@ import (
 	"github.com/openshift/library-go/pkg/controller/factory"
 	"github.com/openshift/library-go/pkg/operator/events"
 	"github.com/openshift/library-go/pkg/operator/resource/resourceapply"
+	"github.com/red-hat-storage/odf-multicluster-orchestrator/controllers/secretutils"
```

Comment:
we have moved the package `secretutils` to `common`.
Functions in are all same in the package.